lib/GADS.pm

```diff
@@ -2892,6 +2891,7 @@ prefix '/:layout_name' => sub {
                 my $checkbox_fields    = [body_parameters->get_all('checkboxes')];
                 my $security_marking   = body_parameters->get('security_marking');
                 my $instance           = $layout->instance_id;
+                my $groups             = body_parameters->get_all('groups');
```

Comment:
This needs to be an array ref, otherwise every other group ID will become a key to this sub